docs/src/user-guide/configuring/configuration-files-new.md

```diff
@@ -561,5 +561,3 @@ npx eslint -c some-other-file.js **/*.js
 ```
 
 In this case, ESLint will not search for `eslint.config.js` and will instead use `some-other-file.js`.
-
-Each configuration file exports one or more configuration object. A configuration object
```

Comment:
Not sure, maybe more lines were meant to be added here?  cc @nzakas

Reply:
It looks like this was copy-pasted from earlier in the page, so it’s safe to remove.